docs/source/checks/nlp/data_integrity/plot_special_characters.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__special_characters:
+
+Special Characters
+******************
+
+This notebook provides an overview for using and understanding the special characters check.
+
+**Structure:**
+
+* `Why check for special characters? <#why-check-for-text-data-duplicates>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
+
+Why check for special characters?
+===================================
+
+The ``SpecialCharacters`` check looks for text sample in which the percentage of special characters
+out of all characters is significant. Such samples can be an indicator for a problem in the data pipeline that
+require attention. Additionally, such examples may be problematic for the model to predict on.
+For example, a text sample with many emojis may be hard to
+predict on and a common mythology will be to replace them with a textual representation of the emoji.
+
+Generate data & model
+=====================
+
+Let's create a simple dataset with some duplicate and similar text samples.
+"""
+
+from deepchecks.nlp.datasets.classification import tweet_emotion
+
+text_data = tweet_emotion.load_data(as_train_test=False)
+text_data.head(3)
+
+#%%
+# Run the Check
+# =============
+
+from deepchecks.nlp.checks import SpecialCharacters
+
+check = SpecialCharacters()
+result = check.run(text_data)
+result.show()
+
+#%%
+# In addition to the check display we can also see receive a summary of most common special characters
```

Comment:
A word about what we're seeing here and in the result

Reply:
added